server/src/main/webapp/WEB-INF/rails/app/controllers/api/stages_controller.rb

```diff
@@ -35,14 +35,6 @@ def index
     end
   end
 
-  def cancel_stage_using_pipeline_stage_name
-    pipeline_name = params[:pipeline_name]
-    stage_name = params[:stage_name]
-    result = HttpLocalizedOperationResult.new
-    schedule_service.cancelAndTriggerRelevantStages(pipeline_name,stage_name, current_user, result)
```

Comment:
This method becomes unused, I think.

Reply:
Yes, this method is no longer used. Removing the same.